examples/lighting-app/telink/Readme.md

```diff
@@ -0,0 +1,180 @@
+![Telink B91 EVK](http://wiki.telink-semi.cn/wiki/assets/Hardware/B91_Generic_Starter_Kit_Hardware_Guide/connection_chart.png)
+
+## Build
+1. Pull docker image from repository:
+    ```
+    $ docker pull alexkolosovtelinksemi/chip-build-telink:latest
```

Comment:
this needs to be part of chip dockerhub - please create a separate PR to setup the image.

Reply:
@andy31415 Could you point me to such PR example?

Reply:
@andy31415, separate pull request has been created for docker image: https://github.com/project-chip/connectedhomeip/pull/6872

Reply:
Can you fix these instructions please?